backend/app/settings.py

```diff
@@ -70,49 +45,59 @@
     },
 ]
 
-WSGI_APPLICATION = "app.wsgi.application"
+# setup the domain to serve from based on environment
+API_DOMAIN = os.environ.get("API_DOMAIN")
+ALLOWED_HOSTS = []
+if API_DOMAIN:
+    ALLOWED_HOSTS.append(API_DOMAIN)
+if DEBUG:
+    ALLOWED_HOSTS.extend(["localhost", "127.0.0.1", "0.0.0.0"])
+
+
+# setup allowed cors origins based on environment
+MAIN_DOMAIN = os.environ.get("MAIN_DOMAIN")
+CORS_ORIGIN_WHITELIST = []
+if MAIN_DOMAIN:
+    CORS_ORIGIN_WHITELIST.append(f"https://{MAIN_DOMAIN}")
+if DEBUG:
+    CORS_ORIGIN_ALLOW_ALL = True
+
+
+MIDDLEWARE = [
+    "django.middleware.security.SecurityMiddleware",
+    "django.contrib.sessions.middleware.SessionMiddleware",
+    "corsheaders.middleware.CorsMiddleware",
+    "django.middleware.common.CommonMiddleware",
+    "django.middleware.csrf.CsrfViewMiddleware",
+    "django.contrib.auth.middleware.AuthenticationMiddleware",
+    "django.contrib.messages.middleware.MessageMiddleware",
+    "django.middleware.clickjacking.XFrameOptionsMiddleware",
+]
 
 
 # Database
 # https://docs.djangoproject.com/en/3.0/ref/settings/#databases
-
 DATABASES = {
     "default": {
-        "ENGINE": "django.db.backends.sqlite3",
-        "NAME": os.path.join(BASE_DIR, "db.sqlite3"),
+        "ENGINE": "django.db.backends.postgresql",
+        "HOST": os.environ.get("POSTGRES_HOST", "localhost"),
```

Comment:
let's be consistent, up above we use named params, so let's update these to:
`os.environ.get("POSTGRES_HOST", default="localhost")`
(and update the others)

Reply:
Updated